src/output.cpp

```diff
@@ -551,7 +551,7 @@ void draw_tabs( const catacurses::window &w, int active_tab, ... )
     }
 
     // Extra "buffer" space per each side of each tab
-    double buffer_extra = ( win_width - total_width ) / ( labels.size() * 2 );
+    double buffer_extra = static_cast<double>( ( win_width - total_width ) / ( labels.size() * 2 ) );
```

Comment:
I don't think this code is doing the right thing.  Everything on the RHS is an integer, so the result is an integer.  Then it gets cast to a `double` and back to an `int`.  Then take the difference between the `double` and `int` values (which will always be zero).